megatron/model/transformer.py

```diff
@@ -471,10 +542,24 @@ def __init__(self, init_method,
         self.core_attention = CoreAttention(self.layer_number,
                                             self.attn_mask_type)
         self.checkpoint_core_attention = args.recompute_granularity == 'selective'
+        
+        self.apply_query_key_layer_scaling = args.apply_query_key_layer_scaling
+        world_size = mpu.get_tensor_model_parallel_world_size()
```

Comment:
tensor_parallel_size

Reply:
> tensor_parallel_size

sorry, I didn't get it. Do you mean the variance name should be tensor_parallel_size？